libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/accounts/Billing.kt

```diff
@@ -0,0 +1,64 @@
+package com.mapbox.services.android.navigation.v5.internal.accounts
+
+import android.content.Context
+import android.content.pm.ApplicationInfo
+import android.content.pm.PackageManager
+import timber.log.Timber
+
+private const val ENABLE_MAU = "EnableMAU"
+private const val META_DATA = "com.mapbox.services.android.navigation.v5"
+private const val ENABLE_MAU_META_DATA = META_DATA + ENABLE_MAU
+
+class Billing private constructor() {
+
+    enum class BillingModel {
+        TRIPS,
+        MAU
+    }
+
+    companion object {
+        private var INSTANCE: Billing? = null
+        private var billingType = BillingModel.TRIPS
+
+        @JvmStatic
+        fun getInstance(context: Context): Billing =
+                INSTANCE ?: synchronized(this) {
```

Comment:
Is there a way to avoid this nested `INSTANCE ?:`?

Reply:
Done